sdk/search/azure-search-documents/CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## 11.1.0b2 (Unreleased)
 
+**Features**
+
+- Added azure.search.documents.RequestEntityTooLargeError
+- Flush method in BatchClient now will not return untill all actions are done
```

Comment:
similarly is there a way to add a test to make sure `flush` is behaving this way?